tools/check_op_desc.py

```diff
@@ -0,0 +1,317 @@
+# Copyright (c) 2019 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import paddle.fluid.framework as framework
+from paddle.fluid import core
+import json
+from paddle import compat as cpt
+import sys
+
+SAME = 0
+
+INPUTS = "Inputs"
+OUTPUTS = "Outputs"
+ATTRS = "Attrs"
+
+ADD = "Add"
+DELETE = "Delete"
+CHANGE = "Change"
+
+DUPLICABLE = "duplicable"
+INTERMEDIATE = "intermediate"
+DISPENSABLE = "dispensable"
+
+TYPE = "type"
+GENERATED = "generated"
+DEFAULT_VALUE = "default_value"
+
+error = False
+
+
+def get_attr_default_value(op_name):
+    return core.get_op_attrs_default_value(cpt.to_bytes(op_name))
+
+
+def get_vars_info(op_vars_proto):
+    vars_info = {}
+    for vat_proto in op_vars_proto:
```

Comment:
There is a typo, `vat_proto` to `var_proto`.

Reply:
Thx. done